tests/integration/test_data_cli.py

```diff
@@ -5,16 +5,59 @@
 
 from planet.cli import cli
 
+TEST_URL = 'http://MockNotRealURL/api/path'
+TEST_DOWNLOAD_URL = f'{TEST_URL}/download'
+TEST_DATA_URL = f'{TEST_URL}/data/v2'
+
+
+@pytest.fixture
+def invoke():
+
+    def _invoke(extra_args, runner=None):
+        runner = runner or CliRunner()
+        args = ['data', '--base-url', TEST_URL] + extra_args
+        return runner.invoke(cli.main, args=args)
+
+    return _invoke
```

Comment:
@jreiberkyle I had a couple thoughts about this fixture while pairing with @mkshah605. First, it hides the `catch_exceptions=False` option of CliRunner.invoke, which is really handy when developing. Second, it puts us in the position of faking the response to a fake URL, which is, in my opinion, one step too far.

I suggest we lean towards configuring the respx mock like `respx.post('https://api.planet.com/data/v1/quick-search').return_value = mock_resp`, using the real endpoint URLs whenever we can. Thoughts?